### ISSUE TITLE
mcapi missing pathlib2 requirement

### ISSUE THREAD

**Issue by bpuchala** [materials-commons/materialscommons.org#1265]
It seems there was a change from 0.7.5 to 0.7.6 to using pathlib2, but the requirements info for pip was not updated.

After updating to materials-commons==0.7.6, I received an error when attempting to use the 'mc' program:
```
$ mc ls .
Traceback (most recent call last):
  File "/anaconda/envs/casm_0.3.X_py36/lib/python3.6/site-packages/materials_commons/__init__.py", line 4, in <module>
    Path().expanduser()
AttributeError: 'PosixPath' object has no attribute 'expanduser'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/anaconda/envs/casm_0.3.X_py36/bin/mc", line 4, in <module>
    import materials_commons.cli as mc_cli
  File "/anaconda/envs/casm_0.3.X_py36/lib/python3.6/site-packages/materials_commons/__init__.py", line 6, in <module>
    from pathlib2 import Path
ModuleNotFoundError: No module named 'pathlib2'
```

```pip install pathlib2``` and then everything seems OK.

**Comment by Terry-Weymouth** [comment]
Yep; apparently you are using Python2, yes?
I’d recommend using Python3 if possible.

I’ll add to the requirements.txt
Thanks for pointing this out.

**Comment by Terry-Weymouth** [comment]
This is done: fixed and tested, for the latest version, 0.7.7b3, and will be released in 0.7.7.
